RocketChat/Rocket.Chat.ReactNative
# Chore: Server API types - user.setPreferences

## Proposed changes
Create type for REST API call: user.setPreferences

## Issue(s)	
N/A

## How to test or reproduce
Open the menu -> profile -> Preferences -> Notifications: 
* Check if every options still works fine;

## Screenshots
N/A

## Types of changes




## Checklist



## Further comments
N/A
